assets/js/components/consent-mode/ConsentModeSetupCTAWidget.js

```diff
@@ -0,0 +1,258 @@
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Fragment,
+	createInterpolateElement,
+	useState,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { Button, SpinnerButton } from 'googlesitekit-components';
+import { CORE_LOCATION } from '../../googlesitekit/datastore/location/constants';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { withWidgetComponentProps } from '../../googlesitekit/widgets/util';
+import { Cell, Grid, Row } from '../../material-components';
+import BannerGraphicsSVG from '../../../svg/graphics/consent-mode-setup.svg';
+import {
+	AdminMenuTooltip,
+	useShowTooltip,
+	useTooltipState,
+} from '../AdminMenuTooltip';
+import ErrorText from '../ErrorText';
+import Link from '../Link';
+import useViewOnly from '../../hooks/useViewOnly';
+import { WEEK_IN_SECONDS } from '../../util';
+import { CONSENT_MODE_SETUP_CTA_WIDGET_SLUG } from './constants';
+
+const { useSelect, useDispatch } = Data;
+
+function ConsentModeSetupCTAWidget( { Widget, WidgetNull } ) {
+	const [ isSaving, setIsSaving ] = useState( false );
+	const [ saveError, setSaveError ] = useState( null );
+
+	const viewOnlyDashboard = useViewOnly();
+
+	const isConsentModeEnabled = useSelect( ( select ) =>
+		select( CORE_SITE ).isConsentModeEnabled()
+	);
+
+	const settingsURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getAdminURL( 'googlesitekit-settings' )
+	);
+
+	const consentModeDocumentationURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getDocumentationLinkURL( 'consent-mode' )
+	);
+
+	const showTooltip = useShowTooltip( CONSENT_MODE_SETUP_CTA_WIDGET_SLUG );
+	const { isTooltipVisible } = useTooltipState(
+		CONSENT_MODE_SETUP_CTA_WIDGET_SLUG
+	);
+
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isPromptDismissed(
+			CONSENT_MODE_SETUP_CTA_WIDGET_SLUG
+		)
+	);
+	const dismissCount = useSelect( ( select ) =>
+		select( CORE_USER ).getPromptDismissCount(
+			CONSENT_MODE_SETUP_CTA_WIDGET_SLUG
+		)
+	);
+
+	const { setConsentModeEnabled, saveConsentModeSettings } =
+		useDispatch( CORE_SITE );
+	const { dismissPrompt } = useDispatch( CORE_USER );
+	const { navigateTo } = useDispatch( CORE_LOCATION );
+
+	if ( isTooltipVisible ) {
+		return (
+			<Fragment>
+				<WidgetNull />
+				<AdminMenuTooltip
+					title=""
```

Comment:
`title` is a required prop, but the Figma design effectively only uses the `content` side of this component. We can take another look at this post-MVP, I don't think it should block us now.